app.js

```diff
@@ -33,7 +40,7 @@ function gameWarriors(){
    return warrior;
 }
 
-
+//make charecter in the DOM
```

Comment:
i dont see the rest of the code did you merged it or pushed it to master?

Reply:
I didnt merge it to the master did I need to ?